app/presenters/manual_presenter.rb

```diff
@@ -14,6 +14,10 @@ def beta?
     true
   end
 
+  def slug_for_search
+    url[1..-1]
```

Comment:
I'm a little worried that this implementation could easily be broken, especially as there don't appear to be any tests. A comment to document the intent would also help.

Finders include the slug in their content item to enable this kind of integration with search - perhaps we should do that for manuals?